### PR TITLE
Redesigns the shuttle bar

### PR DESCRIPTION
**Pull request by Niknakflak**
Redesigns the shuttle bar in okand37's design

New design: 
image

---

Old design:
image

Left the barmaid there despite players consistently adminhelping about them not being able to make drinks.  Planned on doing an icon change over a mob that could but wasn't sure how to go about that to make it functionally the same.  Coiax would also throw a hissy fit if I attempted to do this despite the barmaid still being there, only changed.

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
cozy

**Comment by optimumtact** [comment]
don't you mean the bar shuttle?

**Comment by pudl** [comment]
That wooden floor clashing with the bar looks like a glitch it's hideous.

**Comment by pudl** [comment]
I meant grimy floor.

**Comment by coiax** [comment]
- You removed the pretty cherry tree potted plant
- There's no toxin kit behind the bar anymore
- Why is there a lizard tail drink, get rid of that, bar shuttle caters to all patrons
- It's lost fire extinguishers
- Some of the smoothed walls are a bit weird
- We don't need a full surgery section in the medbay
- There are no sinks or mirrors in the toilets
- In fact, I'm not sure why it has a medbay anyway, you're just cutting down the bathroom space

**Comment by Niknakflak** [comment]
- other plants are pretty too, not my design
- Toxins kit is in the bottom medbay
- was in the design, and has zero impact on gameplay and this is your opinion
- it had 4, now it has 2 plus a shower which can be used
- where
- There's a full surgery section in lots of shuttle medbays, it was part of the admin's design
- Sinks were replaced with a shower, there were never any mirrors in your design to begin with
- oh no a bathroom the horror, it'll be so missed I'm sure.  This is again, an opinion you have about the design which I didn't make.  Please take all grievances to okand37

**Comment by pudl** [comment]
This pr is a direction downgrade in every way.
- There are now a bunch of gross grimy floors that horribly clash with the bar floor
- The old cafeteria table was replaced with a weird kidney table abomination with a comfy chair, this table design makes no sense for any sort of eating place
- Theres a medical booth added when it's mostly a gimmick shuttle
- The bathrooms no longer have sinks
- Theres a billion ugly identical plants
- The security brig has chairs blocking the exit and a redundant table with a single pair of handcuffs. Not even a box of cuffs would justify the uselessness of the table
- The bridge now has a dumbass statue instead of whiskey, which sort of goes against the gimmick of a bar shuttle in favor of a... stupid fucking statue?
- There are now bar stools at a wooden table which is a clash of theme
- Basically everything coiax said

Don't try to reason with niknak anyone, I tried to tell him all of this over steam and he completely ignored me and said it was "just my opinion". :,)

**Comment by optimumtact** [comment]
Well I mean that is correct, it is just your opinion

**Comment by Niknakflak** [comment]
Because a clash of theme is your opinion such as the floors, and the bar stools at wooden tables.

**Comment by coiax** [comment]
The piano/minimog is actually random when it spawns, it turns out

**Comment by coiax** [comment]
Anyway @optimumtact don't merge this until I get some changes made

**Comment by Niknakflak** [comment]
@optimumtact Merge this, coiax can make changes after so I don't have to deal with conflicts

**Comment by coiax** [comment]
I'm not happy with the shuttle bar in this way, so I'd like it not to be merged, until I can finish it.

**Comment by Niknakflak** [comment]
If you fully intend to build upon the changes of this PR, there is zero point in it not being merged

**Comment by coiax** [comment]
I'm not a new coder who can't possibly deal with stuff not being in the master branch, this contains some good ideas, but I don't want it replacing the current shuttle until I make some changes.

**Comment by Niknakflak** [comment]
I don't care what you want.

**Comment by optimumtact** [comment]
Personally I like it
